File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/Specializations.java
package com.BlizzardArmory.model.warcraft.charactersummary;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Specializations.
 */
public class Specializations {

    @SerializedName("href")
    @Expose
    private String href;

    /**
     * Gets href.
     *
     * @return the href
     */
    public String getHref() {
        return href;
    }

    /**
     * Sets href.
     *
     * @param href the href
     */
    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/playerspec/Specializations.kt
package com.BlizzardArmory.model.warcraft.talents.playerspec

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


@Keep
data class Specializations (

    @SerializedName("specialization") val specialization : Specialization,
    @SerializedName("glyphs") val glyphs : List<Glyphs>,
    @SerializedName("pvp_talent_slots") val pvpTalentSlots : List<PvpTalentSlots>,
    @SerializedName("loadouts") val loadouts : List<Loadouts>
)